sql/src/SqlTables.sk

```diff
@@ -616,10 +616,19 @@ fun getIndexByName(
 /* Tables. */
 /*****************************************************************************/
 
+fun getDirDescrs(
+  origContext: readonly SKStore.Context,
+): SortedMap<P.Name, SKDB.DirDescr> {
+  context = SKStore.Context::fromSaved(origContext.clone());
+  getTableDir(context)
+    .items(context)
+    .flatMap(x -> x.i1)
+    .reduce((acc, curr) -> acc.set(curr.name, curr), SortedMap::create())
```

Comment:
[minor] Can you `.collect(SortedMap)`?

Reply:
No FromIterator implementation for SortedMap, no.  Could add that separately but leaving as is for now.